terraform/project.tf

```diff
@@ -0,0 +1,31 @@
+variable "project_name" {}
+variable "billing_account" {}
+variable "region" {}
+
+provider "google" {
+  region = var.region
+}
+
+resource "random_id" "id" {
+  byte_length = 4
+  prefix      = var.project_name
```

Comment:
Randomization is to make the name unique? I'd rather not have random fluff in the names as it's then much harder to remember and type. I think using the "who-myhealth2" as a prefix should suffice. You could declare standard shared config like this in a `terraform.tfvars` file which is automatically loaded anytime Terraform is run.